src/index.css

```diff
@@ -0,0 +1,49 @@
+.h5p-vocabulary-drill {
+  color: #000000;
+  font-family: sans-serif;
+  font-size: 16px;
+  line-height: 1.5em;
+}
+
+.h5p-vocabulary-drill .h5p-question-introduction {
+  border-bottom: 1px solid #eee;
+  margin: 0;
+  padding: 0.8em 1em 1em;
+}
+
+.h5p-vocabulary-drill-toolbar {
+  align-items: center;
+  display: flex;
+  padding: 1em 1em 0;
+  justify-content: space-between;
+}
+
+.h5p-vocabulary-drill-toolbar p {
```

Comment:
Do you think using SCSS instead of CSS would make this easier to read and maintain?

Reply:
Depends on how much styling we are going to add. But yes, in general